contrib/win32/openssh/win32iocompat.vcxproj

```diff
@@ -167,6 +167,9 @@
       <ControlFlowGuard>Guard</ControlFlowGuard>
       <AdditionalOptions>/Gy %(AdditionalOptions)</AdditionalOptions>
     </ClCompile>
+	<Lib>
+      <AdditionalDependencies>Wtsapi32.lib;%(AdditionalDependencies)</AdditionalDependencies>

```

Comment:
Openssh shipped into windows is compiled against onecore.lib. Please check if this code can be compiled with onecore.lib.

Reply:
This dependency is no longer needed. I removed those lines in the latest commit.